docs/getting-started/tutorial-sample-discover.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[tutorial-sample-discover]]
+=== Using Discover
+
+Another way to interactively explore your data is in the Discover application.   
+Here the data is presented as a table of documents.  As in the Dashboard application, 
```

Comment:
I'd avoid leading into the section with "Another..." just because it's so easy to drop into the middle of things. I still twitch every time I see "XYZ application", but I guess I've lost that one. Maybe, "Discover lets you interactively explore the sample data."?  I'm not a fan of "Here...". I'd just say, "The data is presented in a table." Not sure the comparison to the Dashboard application is necessary? I'd probably say "You can search and filter the data..." rather than talking about submitting search queries. In general, we talk about "searching your data" and "submitting queries", I don't think we usually use "search queries" together.